src/sentry/users/models/__init__.py

```diff
@@ -0,0 +1,6 @@
+from ...users.models.users.user import *  # NOQA
```

Comment:
Could we use absolute imports here? The `...` pattern is hard to search for in the future.